search-sentiments-map/routes/search.js

```diff
@@ -186,16 +187,16 @@ async function updateSearchResults() {
   let countries = countriesJson.map(country => country.id);
 
   const trends = await search.retrieveGlobalTrends();
-  for (let i = 0; i < trends.length; i++) {
+  for (let i = 0; i < trends.length; i++) { 
     let topic = trends[i].trendTopic;
     console.log('Creating WorldDataByTopic entity for', topic)
     let countriesData = await search.getSearchResultsForCountriesForTopic(
         countries, topic);
     search.addWorldDataByTopicToDatastore(topic, countriesData);
 
-    // Note: when testing ,use i < 1 to test for only one trend, and comment 
+    // Note: When testing, use i < 1 to test for only one trend, and comment 
     // out `await new Promise` line to avoid 1 minute pauses.
-    await search.sleep(PAUSE_ONE_MIN_MS);
+    // await search.sleep(PAUSE_ONE_MIN_MS);
```

Comment:
```suggestion
    await search.sleep(PAUSE_ONE_MIN_MS);
```
Unsure why this was commented out.